control_plane/src/local_env.rs

```diff
@@ -401,15 +401,18 @@ impl LocalEnv {
 
         fs::create_dir(base_path)?;
 
-        // generate keys for jwt
-        // openssl genrsa -out private_key.pem 2048
+        // Generate keys for jwt
         let private_key_path;
         if self.private_key_path == PathBuf::new() {
             private_key_path = base_path.join("auth_private_key.pem");
+
+            // Generate the public/private key pair
```

Comment:
does it make sense to take the keys (or perhaps only a token signed by a key with this form) and run it through the (real) control plane's tests to help ensure that this remains a reasonable mock?